analyze/analyze.rs

```diff
@@ -801,3 +799,84 @@ pub fn diff(
     let diff = Diff { deltas };
     Ok(Box::new(diff) as Box<traits::Emit>)
 }
+
+#[derive(Debug)]
+struct Garbage {
+    items: Vec<ir::Id>,
+    opts: opt::Garbage,
+}
+
+impl traits::Emit for Garbage {
+    fn emit_text(&self, items: &ir::Items, dest: &mut io::Write) -> Result<(), traits::Error> {
+        let mut table = Table::with_header(vec![
+            (Align::Right, "Bytes".to_string()),
+            (Align::Right, "Size %".to_string()),
+            (Align::Right, "Item".to_string()),
```

Comment:
This column should be titled "Garbage Item" just to help distinguish the output of this command.